omero/developers/build-system.txt

```diff
@@ -165,6 +87,106 @@ of the :model_doc:`OME-Model <ome-xml/>` and generate our |OmeroModel|.
 This code is not available in git, but once built, can be found in all the
 directories named "generated".
 
+Build tools
+-----------
+
+Ant
+^^^
+
+``./build.py`` is a complete replacement for your local ant install. In
+many cases, you will be fine running :program:`ant`. If you have any issues
+(for example ``OutOfMemory``) , please use ``./build.py`` instead. **However,
+only use one or the other; do not mix calls between the two.**
+
+The main build targets are defined in the top-level :file:`build.xml` file.
+All available targets can be listed using::
+
+    ./build.py -p
+
+Each of the component contains a :file:`build.xml` and can be built directly
+using::
+
+    ./build.py -f components/server/build.xml
+
+This will call the default ``dist`` target for each component.
+
+Ivy
+^^^
+
+The build system uses Ivy_ 2.3.0 as the dependency manager. The general Ivy
+configuration is defined in a :ivydoc:`settings file <settings.html>` located
+under :source:`etc/ivysettings.xml`.
+
+In order to determine the transitive closure of all dependencies, Ivy resolves
+each :file:`ivy.xml` file and stores the resolved artifacts in a
+:ivydoc:`cache <settings/caches/cache.html>` to speed up other processes. The
+OMERO build system defines and uses two kinds of caches:
+
+#. a local cache under :file:`lib/cache`
+#. the maven cache under :file:`~/.m2/repository`
+
+.. note::
+
+   When the Ivy configuration file or the version number is changed, the
+   cache can become stale. Calling ``./build.py clean`` from the top-level
+   build will delete the content of the local cache.
+
+:ivydoc:`Resolvers <settings/resolvers.html>` are key to how Ivy functions. Multi-resolvers can be defined (as granular as for an individual jar) in order
```

Comment:
What is meant by 'Multi-resolvers' here? Do you mean 'Multiple resolvers'? And by 'as granular' are you saying fine-grained enough to resolve an individual jar?